hsmd/hsmd_wire.csv

```diff
@@ -323,3 +323,11 @@ msgdata,hsmd_derive_secret,info,u8,len
 # Reply with the derived secret
 msgtype,hsmd_derive_secret_reply,127
 msgdata,hsmd_derive_secret_reply,secret,secret,
+
+# Sanity check this pubkey derivation is correct.
+msgtype,hsmd_check_pubkey,28
+msgdata,hsmd_check_pubkey,index,u32,
+msgdata,hsmd_check_pubkey,pubkey,pubkey,
+
+# Reply (if it's bad, it simple aborts.)
```

Comment:
what does this mean exactly?  closes the connection?  it would be difficult to distinguish from a real disconnect due to restart.

Reply:
OK, this works for a local hsmd, not for remote.  Will make it an explicit reply!

Thanks!